diff -up /home/blp/cs140/pintos/src/lib/debug.c.\~1.8.\~ /home/blp/cs140/pintos/src/lib/debug.c
--- /home/blp/cs140/pintos/src/lib/debug.c.~1.8.~	2004-09-12 13:14:11.000000000 -0700
+++ /home/blp/cs140/pintos/src/lib/debug.c	2004-10-17 00:02:32.000000000 -0700
@@ -5,6 +5,7 @@
 #include <stdio.h>
 #include <string.h>
 #ifdef KERNEL
+#include "threads/init.h"
 #include "threads/interrupt.h"
 #include "devices/serial.h"
 #else
@@ -83,7 +84,7 @@ debug_panic (const char *file, int line,
 
 #ifdef KERNEL
   serial_flush ();
-  for (;;);
+  power_off ();
 #else
   exit (1);
 #endif
