diff -up /home/blp/cs140/bochs-2.2.6/gui/Makefile.in\~ /home/blp/cs140/bochs-2.2.6/gui/Makefile.in
--- bochs-2.2.6.orig/gui/Makefile.in	2004-12-13 11:10:35.000000000 -0800
+++ bochs-2.2.6/gui/Makefile.in	2006-04-03 16:18:10.000000000 -0700
@@ -46,7 +46,7 @@ SHELL = /bin/sh
 CXX = @CXX@
 CXXFLAGS = $(BX_INCDIRS) @CXXFLAGS@  @GUI_CXXFLAGS@
 LOCAL_CXXFLAGS =
-LDFLAGS = @LDFLAGS@
+LDFLAGS = @LDFLAGS@ -lsocket
 LIBS = @LIBS@
 X_CFLAGS = @X_CFLAGS@
 RANLIB = @RANLIB@
